### PR TITLE
pmqa.py:cpufreq feature only supported on PowerNV

### PR DESCRIPTION
**Pull request by shirishaganta**
In PowerVM cpufreq is not supported
so changed the script that cpufreq supports only in PowerNV environment

Signed-off-by: shirisha Ganta <shiganta@in.ibm.com>

### CODE REVIEW

**Comment by shirishaganta** [comment]
Before Patch:
00:47:25 INFO    : Running Host Tests Suite distro_regression_pmqa
00:47:25 INFO    : Running: /usr/bin/avocado run avocado-misc-tests/cpu/pmqa.py --force-job-id 921794bd21e58cd8befe58c3d67566db4667fbee  --job-results-dir /root/avocado-fvt-wrapper/results
JOB ID     : 921794bd21e58cd8befe58c3d67566db4667fbee
JOB LOG    : /root/avocado-fvt-wrapper/results/job-2020-04-25T00.47-921794b/job.log
/usr/lib/python3.6/site-packages/avocado_framework-78.0-py3.6.egg/avocado/plugins/run.py:287: FutureWarning: The following arguments will be changed to boolean soon: sysinfo, output-check, failfast, keep-tmp and ignore-missing-references.
  FutureWarning)
 (1/1) avocado-misc-tests/cpu/pmqa.py:Pmqa.test:  CANCEL: sysfs directory for cpufreq is unavailable. (0.38 s)
RESULTS    : PASS 0 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 1
JOB TIME   : 0.43 s
JOB HTML   : /root/avocado-fvt-wrapper/results/job-2020-04-25T00.47-921794b/results.html

After Patch:
localhost:~/avocado-fvt-wrapper/tests/avocado-misc-tests/cpu # avocado run pmqa.py
/usr/lib/python3.6/site-packages/avocado_framework-78.0-py3.6.egg/avocado/plugins/run.py:287: FutureWarning: The following arguments will be changed to boolean soon: sysinfo, output-check, failfast, keep-tmp and ignore-missing-references.
  FutureWarning)
JOB ID     : 3ba75c335008f145f214ec8acb4f05db0472e870
JOB LOG    : /root/avocado-fvt-wrapper/results/job-2020-04-29T16.35-3ba75c3/job.log
 (1/1) pmqa.py:Pmqa.test: SKIP: This test is not supported on PowerVM platform
RESULTS    : PASS 0 | ERROR 0 | FAIL 0 | SKIP 1 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 0.46 s
JOB HTML   : /root/avocado-fvt-wrapper/results/job-2020-04-29T16.35-3ba75c3/results.html
localhost:~/avocado-fvt-wrapper/tests/avocado-misc-tests/cpu #

File: cpu/pmqa.py [hunk 1]
```diff
@@ -20,9 +20,12 @@
 
 from avocado import Test
 from avocado import main
-from avocado.utils import process, git
+from avocado import skipIf
+from avocado.utils import process, git, cpu
 from avocado.utils.software_manager import SoftwareManager
 
+IS_POWER_NV = 'PowerNV' in cpu._get_cpu_info()
```
Comment (PraveenPenguin) [inline cpu/pmqa.py:27]: I don't think so it working any time as `_get_cpu_info()` only pares first entry as in my system below is the output, can you pleas help me to understand more 

>>> cpu._get_cpu_info()
[b'processor\t: 0\n', b'cpu\t\t: POWER9, altivec supported\n', b'clock\t\t: 2700.000000MHz\n', b'revision\t: 2.2 (pvr 004e 1202)\n']
>>>

**Comment by PraveenPenguin** [comment]
@shirishaganta can you please make travis happy than it is ready to go in

**Comment by shirishaganta** [comment]
> @shirishaganta can you please make travis happy than it is ready to go in

sure will do that

**Pull request merged**